pkg/k8s/apis/cilium.io/client/crds/v2/ciliumbgpadvertisements.yaml

```diff
@@ -239,6 +239,9 @@ spec:
                   required:
                   - advertisementType
                   type: object
+                  x-kubernetes-validations:
```

Comment:
Due to this change, we need to update the CRD schema value.

Reply:
sorry, missed it - updated